src/CoreBundle/Resources/config/project-config_test.yml

```diff
@@ -1,22 +1,23 @@
 imports:
-   - { resource: project-config.yml }
+     - { resource: project-config.yml }
```

Comment:
Some files are renamed to "yaml" in the other PR (https://github.com/chameleon-system/chameleon-system/pull/209)?